galaxyproject/galaxy
# Migrate to MyST-Parser for Markdown docs

replacing deprecated recommonmark.

Fix https://github.com/galaxyproject/galaxy/issues/13005 .

Also:
- Drop not very useful ``doc/source/dev/file_upload.md`` .
- Use auto-generated table of contents in tool schema
- Small doc and link fixes
- Document documentation.

## How to test the changes?
(Select all options that apply)
  1. `GALAXY_DOCS_SKIP_SOURCE=1 make -C doc/ html`
  2. Check the files in doc/build/html/ that are built from `.md` files in `doc/source/`

## License
